pkg/security/util.go

```diff
@@ -2,6 +2,7 @@ package security
 
 const (
 	UIDRangeAnnotation     = "openshift.io/sa.scc.uid-range"
+	FSGroupAnnotation      = "openshift.io/sa.scc.fsgroup"
```

Comment:
yes, except that.  Dang, though I uploaded it last night...I reset my changes this morning to work the SCC strategy side of this.  One sec